src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1356,6 +1417,11 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
         goto err_destroy_dct;
     }
 
+    ucs_assertv((init_attr.cq_len[UCT_IB_DIR_TX] / num_dcis) >=
+                        self->tx.bb_max,
```

Comment:
is it a `(sq_length >= self->tx.bb_max)`?

Reply:
done